docs/tutorials/cli.rst

```diff
@@ -198,9 +198,9 @@ To setup pseudolocalization add :conf:`pseudoLocale` in ``package.json``::
 :conf:`pseudoLocale` option can be any string 
 examples: :conf:`en-PL`, :conf:`pseudo-LOCALE`, :conf:`pseudolocalization` or :conf:`en-UK`
 
-PseudoLocale folder is automatically created based on configuration when running 
-``yarn extract`` command. Pseudolocalized text is created on  ``yarn compile`` command.
-The pseudolocalization is automatically created from default messages. 
+PseudoLocale string have to be in `locale` config as well. Otherwise no folder is going to be created.
+Pseudolocalized text is created on  ``yarn compile`` command.
+The pseudolocalization is automatically created from TODOTODOTODO.
```

Comment:
Do we have already in documentation what is the prioritisation of compiling messages?

Reply:
I don't think so. It's:

1. translation
2. fallback (using the new `fallbackLocales`)
3. message key

Reply:
Wouldn't it be worth adding it to documentation? I just can't figure out where it could fit

Reply:
Yeah, absolutely. Somewhere inside `compile` docs? Because it ultimately happens at compile step.